src/HttpRequest.h

```diff
@@ -285,5 +285,10 @@ void UpdateRequestNotes(ConnStateData *csd, HttpRequest &request, NotePairs cons
 /// nil parameter(s) indicate missing caller information and are handled safely
 const Ip::Address *FindListeningPortAddress(const HttpRequest *, const AccessLogEntry *);
 
+/// \returns listening/*_port port number used by the client connection (or zero)
+/// nil parameter(s) indicate missing caller information and are handled safely
+unsigned short FindListeningPortNumber(const HttpRequest *, const AccessLogEntry *);
+
+
```

Comment:
double empty line.

Reply:
I am surprised you are blocking a PR because of an extra double line, but it is now fixed at [ef62eba](https://github.com/squid-cache/squid/pull/997/commits/ef62ebab2f55687658d94e4eaa3e40247fd2edda).

Reply:
I am surprised you are not aware of standard review procedures. Code style/polish noticed during review always required fixing before merge. Making sure these types of things do not get into the code is one of the maintainer duties you should be performing.

Reply:
IMO, there is no need to block a PR to ask for a new line to be deleted.